README.md

```diff
@@ -95,6 +95,9 @@ best_it_commercetools_filter:
     # Cache life time. Enum Attribute labels are cached to minimize CommerceTools requests.
     # DEPRECATED: Do not use. Switch to enum_normalizer
     cache_life_time:      86400
+    
+    # Optional query for defining a base category. If used, only products under the base category will be selected. (default: null)
+    base_category_query: 'custom(fields(alias="_SHOP_ROOT"))'
```

Comment:
Auf Kategorien gibt es ja mittlerweile auch einen key ;)

Reply:
Ist ja lediglich ein Query Beispiel für die README. In den jeweiligen Projekten kann die Query ja auf den Key gehen anstatt einem Custom Field.  ;-)